git-hub

```diff
@@ -81,21 +137,325 @@ def push():
     "git push user pr/num:branch"
 
     """
+    p = subprocess.Popen("git branch", stdout=subprocess.PIPE)
+    pr = str(p.stdout.read())
+    pr = pr.split("*", 1)[1].split()[0]
+    pr = pr.replace("\\n", "").replace(" ", "").replace("'", "")
     try:
-        with open(".git_hub_cache") as json_file:
-            data = json.load(json_file)
-            other_user = data["user"]
-            pr = data["pr"]
-            branch = data["branch"]
-            p = subprocess.Popen(["git", "push", other_user,
-                                 f'{pr}:{branch}'], stdout=subprocess.PIPE)
-            p.stdout.read()
+        file_path = path_to_git()
+        file_path = os.path.join(file_path, '.git_hub_cache')
+        f = open(file_path, "r")
+        pr_dict = toml.load(f)
+        user = pr_dict[pr]['user']
+        branch = pr_dict[pr]["branch"]
+        p = subprocess.Popen(["git", "push", user,
+                             f'{pr}:{branch}'], stdout=subprocess.PIPE)
+        p.stdout.read()
+        print(["git", "push", user, f'{pr}:{branch}'])
     except FileNotFoundError as e:
         print(e)
         print("This command must be used after git hub pr.")
         sys.exit(1)
 
 
+def path_to_git():
+    path_repo = os.path.abspath('.')
+    while os.path.abspath(path_repo) != '/' and not os.path.isdir(pjoin(path_repo, '.git')):  # if not in directory with .git, keep going back to find file
+        path_repo = pjoin(path_repo, '..')
+    path_git = pjoin(path_repo, ".git")
+    return path_git
+
+
+def find_in_dictionary(pr_num, open_or_closed_dictionary):
+    """Fetches all the information on a certain PR
+
+    Parameters
+    ----------
+    pr_num : integer
+        number of PR to fetch information from
```

Comment:
Please capitalize docstrings (i.e., capital first word, full-stop at end of sentence).